scaleway/resource_lb_route_test.go

```diff
@@ -39,12 +39,58 @@ func TestAccScalewayLbRoute_Basic(t *testing.T) {
 					resource scaleway_lb_route rt01 {
 						frontend_id = scaleway_lb_frontend.frt01.id
 						backend_id = scaleway_lb_backend.bkd01.id
-						match_sni = "scaleway.com"
+						match {
+							sni = "sni.scaleway.com"
+						}  
 					}
 				`,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckScalewayLbRouteExists(tt, "scaleway_lb_route.rt01"),
-					resource.TestCheckResourceAttr("scaleway_lb_route.rt01", "match_sni", "scaleway.com"),
+					resource.TestCheckResourceAttr("scaleway_lb_route.rt01", "match.0.sni", "sni.scaleway.com"),
```

Comment:
Please use our domain test

Reply:
IMO we don't really need to use a valid domain here as the backend doesn't verify if the value provided is legit or not